src/components/ui/Panel.jsx

```diff
@@ -188,7 +184,10 @@ export default class Panel extends React.Component {
       >
         {resizable && size === 'minimized' && minimizedTitle ? minimizedTitle : title}
       </div>
-      <div className="panel-content">
+      <div className="panel-content"
+        style={(size === 'default' ? { overflow: 'hidden' } : {})}
```

Comment:
`size` is set as a class on panel, so you should be able to set this rule in the CSS with `.panel.default`. (BTW, I should have written these clases as `panel--{size}`, but that's another problem)

Reply:
done, thanks